src/Microsoft.AspNetCore.Server.IISIntegration/WebHostBuilderIISExtensions.cs

```diff
@@ -52,6 +52,7 @@ public static IWebHostBuilder UseIISIntegration(this IWebHostBuilder hostBuilder
                 {
                     // Delay register the url so users don't accidently overwrite it.
                     hostBuilder.UseSetting(WebHostDefaults.ServerUrlsKey, address);
+                    hostBuilder.PreferHostingUrls(true);
```

Comment:
It wouldn't belong in the template. You only want it active when the ANCM environment variables trigger it. Otherwise it no-op.

Also, we're taking UseIISIntegration out of the template.